tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('build-deploy:devapp', ['build:devapp'], () => {
```

Comment:
Named it `prepare-deploy:devapp` for now. Let's wait for @jelbourn's feedback before I apply merge ready.